packages/teams-js/src/internal/communication.ts

```diff
@@ -565,7 +565,15 @@ async function shouldProcessMessage(messageSource: Window, messageOrigin: string
   ) {
     return true;
   } else {
-    const isOriginValid = await validateOrigin(new URL(messageOrigin));
+    let messageOriginURL: URL;
+    try {
+      messageOriginURL = new URL(messageOrigin);
+    } catch (_) {
+      shouldProcessMessageLogger('Message has an invalid origin of %s', messageOrigin);
```

Comment:
Can we get a unit test added to `communication.spec.ts` that hits this catch block?

Reply:
Added